### DISCUSSION TITLE
Clarification on scraping endpoints

### DISCUSSION THREAD

**Discussion by reubs-ww** [davidteather/TikTok-Api#1061] [Q&A]
Hey, I notice that most of the URL being scraped is `tiktok.com/api` and not the official api.tiktok.com or whatever it is now. Are there any docs revolving aroudn that /api endpoint? I cant seem to find anything on it online. How did you guys find out to scrape from that endpoint and how did you guys find out the other paths after /api?

**Comment by reubs-ww** [comment]
Also, how to know what are the query params required? other then the ones in this lib. I keep getting input error when I try some other endpoints thats not in this lib

**Comment by Charlotte-br560** [comment]
Hey there! We found that scraping from the /api endpoint on tiktok.com provided the data we needed, and it seemed to be effective for our purposes. Unfortunately, there aren't official docs for that specific endpoint, but through some exploration and testing, we identified the paths we needed after /api. If you're looking for a more structured approach, you might want to check out [Crawlbase](https://crawlbase.com/)—it's a tool that could help streamline the process. Cheers!
